rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChangeSetExt.java

```diff
@@ -173,7 +181,7 @@ protected void mapFields(ChangeLogSet<? extends ChangeLogSet.Entry> changeset, W
             commit.setCommitId(entry.getCommitId());
             commit.setCommitUrl(repoUrl);
             commit.setMessage(entry.getMsg());
-            commit.setAuthorJenkinsId(entry.getAuthor().getFullName());
+            commit.setAuthorJenkinsId(isResolvingCommitAuthors() ? entry.getAuthor().getFullName() : "Unidentified");
```

Comment:
Then use `""`, which should not break any code, and have the UI skip display of empty strings.